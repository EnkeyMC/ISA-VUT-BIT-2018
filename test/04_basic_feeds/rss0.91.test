NAME=Basic RSS 0.91 parsing
ARGS=http://www.mocky.io/v2/5bd304933400004e00cfdfe3
RC=0
CHECK_OUTPUT=1
