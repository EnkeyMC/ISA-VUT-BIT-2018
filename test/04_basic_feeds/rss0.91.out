*** WriteTheWeb ***
Giving the world a pluggable Gnutella
Syndication discussions hot up
Personal web server integrates file sharing and messaging
Syndication and Metadata
UK bloggers get organised
Yournamehere.com more important than anything
